src/hooks/useBookSearch.jsx

```diff
@@ -10,7 +15,7 @@ export default function useBookSearch(query, orderBy, pageNumber, setAreResultsL
 
   useEffect(() => {
     if (query === undefined) {
-      setError(""); //Display no error on first page load
```

Comment:
Without this comment it would be difficult for someone figure out why this code exists in the file.

Reply:
reinstated 👍